locksmith/src/utils/middlewares/recaptchaMiddleware.ts

```diff
@@ -2,11 +2,25 @@ import { RequestHandler } from 'express'
 import fetch from 'isomorphic-fetch'
 import config from '../../config/config'
 
+/**
+ * A list of authenticated users who are making calls for which we should bypass the captcha veification
+ */
+const allowList = ['0x61be315032235Ac365e39705c11c47fdaee698Ee'].map(
+  (address: string) => address.toLowerCase()
+)
+
```

Comment:
Probably better to put this in the database directly so we can modify this on the fly.

Reply:
You are right. I'll do it.